src/components/views/dialogs/SetMxIdDialog.js

```diff
@@ -267,24 +267,21 @@ export default React.createClass({
                     </div>
                     { usernameIndicator }
                     <p>
-                        { _tJsx(
+                        { _t(
                             'This will be your account name on the <span></span> ' +
                             'homeserver, or you can pick a <a>different server</a>.',
-                            [
-                                /<span><\/span>/,
-                                /<a>(.*?)<\/a>/,
-                            ],
-                            [
-                                (sub) => <span>{ this.props.homeserverUrl }</span>,
-                                (sub) => <a href="#" onClick={this.props.onDifferentServerClicked}>{ sub }</a>,
-                            ],
+                            {},
+                            {
+                                'span': () => <span>{ this.props.homeserverUrl }</span>,
```

Comment:
Good catch, but I think it might be better to save for later. Changing it to a normal variable would require changing it in every translation as well. which would make this PR balloon even more (or force every translator to re-translate the string).

Reply:
Fair enough